src/jobsautoreport/report.py

```diff
@@ -5,22 +5,26 @@
 
 from jobsautoreport.models import JobState, JobType, StepState
 from jobsautoreport.query import Querier
-from prowjobsscraper.event import JobDetails, StepEvent
+from prowjobsscraper.event import JobDetails
 
 
 class Report(BaseModel):
 
     number_of_e2e_or_subsystem_periodic_jobs: int
     number_of_successful_e2e_or_subsystem_periodic_jobs: int
     number_of_failing_e2e_or_subsystem_periodic_jobs: int
-    success_rate_for_e2e_or_subsystem_periodic_jobs: float
-    top_10_failing_e2e_or_subsystem_periodic_jobs: list[tuple[str, float]]
+    success_rate_for_e2e_or_subsystem_periodic_jobs: tuple[int, int, float]
```

Comment:
In this context it's hard to understand what ``tuple[int, int, float]`` represents (what the first integer means?  what the next integer means? etc.)

Create a class to have it better annotated. E.g.:
```
@dataclasses.dataclass
class Something:
    metric1: int
    metric2: int
    metric3: float
```

Change also ``_get_success_rate`` to use this class